### DISCUSSION TITLE
[Component Idea] Rich Text Editor

### DISCUSSION THREAD

**Discussion by rawilk** [rawilk/laravel-form-components#5] [Ideas]
Need to find a rich text editor I like that can be implemented as a blade/alpine component.

**Comment by mertasan** [comment]
If you want, I can create a PR by adapting the [Trix Editor](https://trix-editor.org) or [Froala Editor](https://froala.com/) components I have prepared before to this package.

(Froala Editor is a paid editor. [ref](https://froala.com/wysiwyg-editor/pricing/))

**Comment by rawilk** [comment]
PR #53 will add a rich text editor component to the package. The editor will be using [Quill](https://quilljs.com) behind the scenes.

I'm not sure if I'll add other rich text editors in the future, but there'll be at least this one in the package for now.
